aea/protocols/fipa/README.md

```diff
@@ -0,0 +1,6 @@
+# fipa protocol
+
+## To update the `fipa_pb2.py` file
+
+	  cd aea/protocols/fipa
+	  protoc --python_out=. fipa.proto
```

Comment:
Some words on `protoc` installation? E.g. links to the official website, a list of commands? 
Maybe not the right place for that. 
Have a look [here](https://github.com/fetchai/z-oef-sdk-python-v1#dependencies) for an example of what I mean.